go/vt/vtgate/endtoend/main_test.go

```diff
@@ -166,6 +166,18 @@ var (
 					Type: sqltypes.Char,
 				}},
 			},
```

Comment:
so, I know it's not clear, but we are moving away from writing these tests here. Instead, we want to use `go/test/endtoend/vtgate` instead. 

The tests in `go/vt/vtgate/endtoend` use the `vtestserver` for testing, whereas `go/test/endtoend/vtgate` uses a proper cluster for testing, and we much prefer that way of testing.

Sorry about not making this clearer in the code today